packages/runtime/container-runtime/src/summaryManager.ts

```diff
@@ -48,7 +43,60 @@ type ShouldSummarizeState =
     | { shouldSummarize: true; shouldStart: boolean; }
     | { shouldSummarize: false; stopReason: StopReason; };
 
-export class SummaryManager extends EventEmitter implements IDisposable {
+export async function requestSummarizer(loader: ILoader, sequenceNumber: number): Promise<ISummarizer> {
+    // TODO eventually we may wish to spawn an execution context from which to run this
+    const request: IRequest = {
+        headers: {
+            [LoaderHeader.cache]: false,
+            [LoaderHeader.clientDetails]: {
+                capabilities: { interactive: false },
+                type: summarizerClientType,
+            },
+            [DriverHeader.summarizingClient]: true,
+            [LoaderHeader.reconnect]: false,
+            [LoaderHeader.sequenceNumber]: sequenceNumber,
+        },
+        url: "/_summarizer",
+    };
+
+    const response = await loader.request(request);
```

Comment:
Any chance you can use requestFluidObject() instead? fluid/component is gone from repo. And ideally we use unified reporting in this space (that logs as much as possible from privacy POV - there are some active bugs in this area for that flow).